packages/react-core/src/next/components/Wizard/examples/WizardValidateOnButtonPress.tsx

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+
+import {
+  Button,
+  Alert,
+  EmptyState,
+  EmptyStateIcon,
+  Title,
+  EmptyStateBody,
+  Progress,
+  EmptyStateSecondaryActions,
+  Form,
+  FormGroup,
+  TextInput
+} from '@patternfly/react-core';
+import { Wizard, WizardStep, WizardFooterWrapper, useWizardContext } from '@patternfly/react-core/next';
+import CogsIcon from '@patternfly/react-icons/dist/esm/icons/cogs-icon';
+
+interface ValidationProgressProps {
+  onClose(): void;
+}
+
+const ValidationProgress: React.FunctionComponent<ValidationProgressProps> = ({ onClose }) => {
+  const [percentValidated, setPercentValidated] = React.useState(0);
+
+  const tick = React.useCallback(() => {
+    if (percentValidated < 100) {
+      setPercentValidated(prevValue => prevValue + 20);
+    }
+  }, [percentValidated]);
+
+  React.useEffect(() => {
+    const interval = setInterval(() => tick(), 1000);
+
+    return () => {
+      clearInterval(interval);
+    };
+  }, [tick]);
+
+  return (
+    <div className="pf-l-bullseye">
+      <EmptyState variant="large">
+        <EmptyStateIcon icon={CogsIcon} />
+        <Title headingLevel="h4" size="lg">
+          {percentValidated === 100 ? 'Validation complete' : 'Validating credentials'}
+        </Title>
+        <EmptyStateBody>
+          <Progress value={percentValidated} measureLocation="outside" aria-label="Wizard validation progress" />
+        </EmptyStateBody>
+        <EmptyStateBody>
+          Description can be used to further elaborate on the validation step, or give the user a better idea of how
+          long the process will take.
+        </EmptyStateBody>
+        <EmptyStateSecondaryActions>
+          <Button isDisabled={percentValidated !== 100} onClick={onClose}>
+            Log to console
+          </Button>
+        </EmptyStateSecondaryActions>
+      </EmptyState>
+    </div>
+  );
+};
+
+interface LastStepFooterProps {
+  isValid: boolean;
+  hasErrorOnSubmit: boolean;
+  setIsSubmitted(isSubmitted: boolean): void;
+  setHasErrorOnSubmit(isSubmitted: boolean): void;
+}
+
+const LastStepFooter: React.FunctionComponent<LastStepFooterProps> = ({
+  isValid,
+  hasErrorOnSubmit,
+  setIsSubmitted,
+  setHasErrorOnSubmit
+}) => {
+  const { goToStepByName, goToNextStep } = useWizardContext();
+
+  const validateLastStep = (onNext: () => void) => {
+    setIsSubmitted(true);
+
+    if (!isValid) {
+      setHasErrorOnSubmit(true);
+      setIsSubmitted(false);
+    } else {
+      onNext();
+    }
+  };
+
+  return (
+    <>
+      {hasErrorOnSubmit && <Alert isInline variant="danger" title="Validation failed. Go back to the Step 1." />}
```

Comment:
Placing an element here isn't supported by the component. It works, but I don't think we should recommend it. As far as putting custom stuff in the wizard elements, it should either go in the body or the footer, though you can't put it in the footer since it's only designed to hold actions. We could update it to support messaging like this (or to support whatever content you want in the footer) if we need to.

<img width="1221" alt="Screenshot 2023-01-18 at 11 26 37 AM" src="https://user-images.githubusercontent.com/35148959/213252388-51f1fe0a-afb1-4baa-a5eb-250094e89b6f.png">

Also it seems like this error would go at the top per the [alert design guidelines](https://www.patternfly.org/v4/components/alert/design-guidelines#using-bordered-inline-alerts)? There is an example of an inline alert at the bottom of the wizard content in the design guidelines, but it's for when the alert only pertains to a specific part of the content - I believe the suggestion is to put the alert inline with the content it pertains to. @mceledonia @mcarrano @mmenestr is that right?

I think the existing/non-next example is how I would expect it, but we should be using an inline error (like you've done it in this PR) instead of a toast warning alert. Here's the existing example:

<img width="854" alt="Screenshot 2023-01-18 at 11 32 01 AM" src="https://user-images.githubusercontent.com/35148959/213253726-3fd481df-751e-4ed7-8b03-fc460eaa937c.png">

Reply:
Admittedly, I did try to rework the existing example here to be a little more intuitive and useful. I felt as though the "Go to Beginning" button had no meaning for the example, so I moved the form input to step 1, and in doing so, felt like an Alert would be useful to show an error on the final step, where now "Going to Beginning" is necessary to correct the validation issue.

For now, I've moved the form input back to the final step, am showing the warning alert above the form on "Validate" click if the input is invalid, removed the status icon for step 1 and the error alert that used to show above the footer for the final step.

Reply:
I chatted with @mcarrano and confirmed it should be an inline error alert instead of a non-inline (toast) warning alert. The positioning is fine. Though it's not a huge deal, so no worries if we want to merge this as-is. But if it's an easy update, we could update that here and in the non-next example.

Also not a blocker but a nice to have, the actions could use updates - though they conflict with one another, depending on the wording we want to use:
* We shouldn't have 2 primary buttons in a row like that. "Go to beginning" could be a secondary button.
* "Go to Beginning" should be sentence case "Go to beginning".
* Unless there's a reason to take someone to the start of the form, "Go to beginning" could just be "Back" like the rest of the buttons.
* Do you know why these actions use "Forward" and "Backward" instead of "Next" and "Back" like the other examples?